### PR TITLE
Update ASP.NET Core Get Started for 3.0

### LINKED ISSUE #10479: Update Get Started tutorial for 3.0
Update the tutorial for ASP.NET Core 3.0.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ca6139d4-022c-395a-0abf-539660550df1
* Version Independent ID: 3dd23f8a-300e-3f78-1dc5-8b759aabdccc
* Content: [Get started with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-2.1&tabs=windows#next-steps)
* Content Source: [aspnetcore/getting-started/index.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/getting-started/index.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by wadepickett**
Tested with 3.0.  
Updated the SDK version for the prerequisite
Added a brief explanation for dotnet new command params.
Fixes #10479

### CODE REVIEW

**Comment by wadepickett** [comment]
Fixes #10479

File: aspnetcore/getting-started/index.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ Open a command shell, and enter the following command:
 dotnet new webapp -o aspnetcoreapp
 ```
 
+The preceding command creates a new application of type `webapp` for you.  The `-o` parameter creates a directory named *aspnetcoreapp* where your app is stored, and populates it with the required files.
```
Comment (Rick-Anderson) [inline aspnetcore/getting-started/index.md]: The preceding command:

* Creates a new web app.
* The `-o` parameter creates a directory named *aspnetcoreapp* with the source files for the app.

Comment (Rick-Anderson) [reply aspnetcore/getting-started/index.md]: I don't allow you/your unless it makes the statement more clear. In most cases (like this), it just adds words.

Comment (Rick-Anderson) [reply aspnetcore/getting-started/index.md]: `@tdykstra` is on vacation.

Comment (Rick-Anderson) [reply aspnetcore/getting-started/index.md]: `@tdykstra` is on .NET now

Comment (wadepickett) [reply aspnetcore/getting-started/index.md]: @Rick-Anderson, Thanks for the guidance! Will move to ga, follow "you/your" guideline.  I thought Tom and I might still review each others PR's.  I didn't realize he was on vacation, thanks.  Will set reviews to others on ASP.NET sub-team members going forward.

**Pull request merged**